src/Aquifer.Data/Entities/VersificationExclusionEntity.cs

```diff
@@ -0,0 +1,17 @@
+using Aquifer.Data.EventHandlers;
+
+namespace Aquifer.Data.Entities;
+
+public class VersificationExclusionEntity : IHasUpdatedTimestamp
+{
+    public int Id { get; set; }
+    public int BibleId { get; set; }
+    public int BibleVerseId { get; set; }
+    public BibleEntity Bible { get; set; } = null!;
+    public VerseEntity Verse { get; set; } = null!;
+    [SqlDefaultValue("getutcdate()")]
+    public DateTime Created { get; set; } = DateTime.UtcNow;
+
+    [SqlDefaultValue("getutcdate()")]
+    public DateTime Updated { get; set; } = DateTime.UtcNow;
```

Comment:
I don't think these are going to get updated really. Don't think we need this on either one.